src/util/Layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Text } from '@ui5/webcomponents-react/lib/Text';
+
+export const TextWrapper = ({ children, isOnHeader, style }) => {
+  return (
+    <Text
+      tooltip={children}
+      wrapping
+      className={isOnHeader && 'text-header-color'}
```

Comment:
Maybe I can get rid of this unused class for headers, right?

Reply:
no css i think
only style up there in the component